python/flink_agents/api/vector_stores/vector_store.py

```diff
@@ -176,3 +250,74 @@ def query_embedding(self, embedding: List[float], limit: int = 10, **kwargs: Any
         Returns:
             List of documents matching the search criteria
         """
+
+    @abstractmethod
+    def add_embedding(
```

Comment:
`add_embedding` and `query_embedding` are not meant for users to call. We probably should mark them as protected.